src/bundle/Resources/config/routing.yml

```
@@ -51,6 +51,15 @@ ibexa.rest.delete_section:
 # Content
 
 
+ibexa.rest.content.create_draft_from_current_version:
+    path: /content/objects/{contentId}/currentversion
```

I am not familiar with Open API specs but would it work (more readable)?
```suggestion
    path: /content/objects/{contentId}/current_version
```

---

Looks better but I wonder if that wouldn't cause additional confusion with the old endpoint? in other related PRs we leave the paths the same as before so this could be a bit misleading for a single endpoint.